polyglot/piranha/Cargo.toml

```diff
@@ -1,12 +1,15 @@
 [package]
-authors = ["Ameya Ketkar <ketkara@uber.com>", "Murali Krishna Ramanathan <murali@uber.com>"]
+authors = ["Ameya Ketkar <ketkara@uber.com>", "Lazaro Clapp <lazaro@uber.com>", "Murali Krishna Ramanathan"]
```

Comment:
Bit of an exaggeration, since I haven't contributed any code to PolyglotPiranha, but I guess code reviews sort of count (plus the original PiranhaJava)...

Reply:
Also, unless there is an explicit organization field, maybe "Uber Technologies Inc." should be here at the end?

Reply:
Will add Uber to author list - [Ref](https://doc.rust-lang.org/cargo/reference/manifest.html#the-authors-field)